### PR TITLE
docs(app): webpack to vite

### PR DESCRIPTION
**Pull request by koji**
# Overview
change webpack to vite and remove unused package description

close AUTH-261

# Test Plan

# Changelog

# Review requests

# Risk assessment
low

### CODE REVIEW

File: app/README.md [hunk 1]
```diff
@@ -131,10 +130,9 @@ ANALYZER=1 make -C app
 [api-server-source]: ../api/opentrons/server
 [electron]: https://www.electronjs.org/
 [electron-renderer]: https://electronjs.org/docs/tutorial/quick-start#renderer-process
-[hmr]: https://webpack.js.org/concepts/hot-module-replacement/
 [react]: https://react.dev/
 [redux]: http://redux.js.org/
 [css-modules]: https://github.com/css-modules/css-modules
 [babel]: https://babeljs.io/
-[webpack]: https://webpack.js.org/
+[viete]: https://vitejs.dev/
```
Comment (ecormany) [inline app/README.md]:
```suggestion
[vite]: https://vitejs.dev/
```
Link was broken because of typo.

**Review by ecormany — COMMENTED** [review]
Need to fix broken markdown anchor. Otherwise good.

File: app/README.md (hunk 1 above)
Comment (koji) [reply app/README.md]: thank you!

**Review by ecormany — APPROVED** [review]
:shipit:

**Pull request merged**